test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Listener/ResponseBodyTests.cs

```diff
@@ -492,7 +492,7 @@ public class ResponseBodyTests
                 await Assert.ThrowsAsync<TaskCanceledException>(() => responseTask);
                 Assert.True(context.DisconnectToken.WaitHandle.WaitOne(TimeSpan.FromSeconds(5)));
                 // It can take several tries before Write notices the disconnect.
-                for (int i = 0; i < 100; i++)
+                for (int i = 0; i < 1000; i++)
```

Comment:
These aren't strictly necessary but even in the same test file, I see 10, 100, 1000 and I just thought it'd be nice to use the same limit. I can extract it to a static readonly if we want.

Reply:
I think it should be fine this way.

Reply:
I think it would be good cleanup to make these a static, but I wouldn't hold up the PR for it.